### PR TITLE
Refs #6558 - show terminated employees with future DOT

### PR DESCRIPTION
**Pull request by je110**
### Redmine ticket(s)
https://devops.dchbx.org/redmine/issues/6558

### Local build result

```
bundle exec rake parallel:spec[4]
913 examples, 0 failures, 18 pending
3967 examples, 0 failures, 78 pendings

bundle exec cucumber
48 scenarios (48 passed)
916 steps (916 passed)
```

### Latest rebase/merge tag
*

---

### Data ticket(s) Followup
* (redmine links here - optional)

### Related Pull Requests
* (github links here - optional)

---

#### Peer Review
* (For code review)

#### Functional Testing
* (For testing locally)

#### Deployment
* (for release manager and/or build manager)

### CODE REVIEW

File: lib/tasks/hbx_reports/employee_terminations.rake [hunk 1]
```diff
@@ -6,9 +6,7 @@ namespace :reports do
     desc "Employee terminations by employer profile and date range"
     task :employee_terminations => :environment do
 
-      date_range = Date.new(2015,10,1)..TimeKeeper.date_of_record
-
-      census_employees = CensusEmployee.find_all_terminated(date_range: date_range)
+      census_employees = CensusEmployee.unscoped.terminated.where(:employment_terminated_on.gte => Date.new(2015,10,1))
```
Comment (davidplappert) [inline lib/tasks/hbx_reports/employee_terminations.rake]: should we use timekeeper?

Comment (je110) [inline lib/tasks/hbx_reports/employee_terminations.rake]: It doesn't need an end date anymore

Comment (davidplappert) [inline lib/tasks/hbx_reports/employee_terminations.rake]: so ... should we remove that line?

Comment (je110) [inline lib/tasks/hbx_reports/employee_terminations.rake]: which line? My new line is already without an end date

**Pull request merged**